spec.html

```diff
@@ -7628,19 +7627,44 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     </ul>
 
     <emu-note>
-      <p>Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.</p>
+      Host environments are not required to treat Jobs uniformly with respect to scheduling. For example, web browsers and Node.js treat Promise-handling Jobs as a higher priority than other work; future features may add Jobs that are not treated at such a high priority.
     </emu-note>
 
+    <p>A Job _job_ <dfn id="job-propagatesactivescriptormodule">propagates the script or module</dfn> _scriptOrModule_ (a Script Record, a Module Record, or *null*) if all of the following conditions are true whenever _job_ is called:</p>
```

Comment:
It's not clear to me whether "whenever _job_ is called" means here. A Job is just an abstract closure. It looks like this is trying to make an assertion about the ways in which the Job is invoked, but that's not really a property of the Job itself (as currently described).

Perhaps it would be better to speak of "an invocation of a Job" here? (If necessary, we could define that term above, by appending to "Such operations accept a Job Abstract Closure as the parameter and schedule it to be performed at some future time" the phrase "; this is called an invocation of the job".)

Reply:
Ended up rephrasing step 2 as "Invoke the Job Abstract Closure" from "Call the Abstract Closure". The definition of Abstract Closure already talks about invocations, so I don't think we need to further define what "invocation of a job" means.